signup/tool/src/webapp/js/signupScript.js

```diff
@@ -762,8 +762,24 @@ function displayDateTime(d){
 	var curr_date = d.getDate();	
 	var curr_month = d.getMonth();
 	var curr_year = d.getFullYear();	
-	return (curr_hour + ":" + curr_min + " " + a_p + ", " + d_names[curr_day] + ", "
+
+	var lang_es = document.querySelector("html").getAttribute("lang");
+
+	if (lang_es == 'es-ES'){

```

Comment:
While this provides support for es-ES, what about all other non-default locales? This approach doesn't seem ideal to provide real i18n support.

Reply:
portal.locale contains the logged-in user's locale

date.toLocaleDateString
date.toLocaleTimeString

look like they can accomplish what you need

Reply:
Thanks @ottenhoff! I'll take a look.

Reply:
@bjones86 @ottenhoff Fixed!